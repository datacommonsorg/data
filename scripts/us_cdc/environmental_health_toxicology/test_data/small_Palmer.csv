year,month,statefips,countyfips,pdsi
1895,1,1,1001,0.64
1896,1,1,1001,-2.05
1897,1,1,1001,-3.76
1898,1,1,1001,-3.77
1899,1,1,1001,-2.52
1900,1,1,1001,-3.17
1901,1,1,1001,-0.4
1902,1,1,1001,0.69
1903,1,1,1001,-0.29
1904,1,1,1001,-2.25
1905,1,1,1001,0.27
1906,1,1,1001,-0.61
1907,1,1,1001,-1.42
1908,1,1,1001,0.1
1909,1,1,1001,-2.17
1910,1,1,1001,-2.46
1911,1,1,1001,-1.42
1912,1,1,1001,1.29
1913,1,1,1001,2.41
