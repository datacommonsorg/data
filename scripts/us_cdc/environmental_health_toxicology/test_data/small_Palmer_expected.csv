year,month,date,statefips,countyfips,StatisticalVariable,Value,dcid
1895,01,1895-01,1,01001,PalmerDroughtSeverityIndex,0.64,geoId/01001
1896,01,1896-01,1,01001,PalmerDroughtSeverityIndex,-2.05,geoId/01001
1897,01,1897-01,1,01001,PalmerDroughtSeverityIndex,-3.76,geoId/01001
1898,01,1898-01,1,01001,PalmerDroughtSeverityIndex,-3.77,geoId/01001
1899,01,1899-01,1,01001,PalmerDroughtSeverityIndex,-2.52,geoId/01001
1900,01,1900-01,1,01001,PalmerDroughtSeverityIndex,-3.17,geoId/01001
1901,01,1901-01,1,01001,PalmerDroughtSeverityIndex,-0.4,geoId/01001
1902,01,1902-01,1,01001,PalmerDroughtSeverityIndex,0.69,geoId/01001
1903,01,1903-01,1,01001,PalmerDroughtSeverityIndex,-0.29,geoId/01001
1904,01,1904-01,1,01001,PalmerDroughtSeverityIndex,-2.25,geoId/01001
1905,01,1905-01,1,01001,PalmerDroughtSeverityIndex,0.27,geoId/01001
1906,01,1906-01,1,01001,PalmerDroughtSeverityIndex,-0.61,geoId/01001
1907,01,1907-01,1,01001,PalmerDroughtSeverityIndex,-1.42,geoId/01001
1908,01,1908-01,1,01001,PalmerDroughtSeverityIndex,0.1,geoId/01001
1909,01,1909-01,1,01001,PalmerDroughtSeverityIndex,-2.17,geoId/01001
1910,01,1910-01,1,01001,PalmerDroughtSeverityIndex,-2.46,geoId/01001
1911,01,1911-01,1,01001,PalmerDroughtSeverityIndex,-1.42,geoId/01001
1912,01,1912-01,1,01001,PalmerDroughtSeverityIndex,1.29,geoId/01001
1913,01,1913-01,1,01001,PalmerDroughtSeverityIndex,2.41,geoId/01001
